jenkins-pipelines/operator/upgrade/upgrade-scylla-k8s-eks.jenkinsfile

```diff
@@ -4,10 +4,13 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingOperatorUpgradePipeline(
     backend: 'k8s-eks',
-    region: 'eu-north-1',
-    base_versions: '["2022.1.6"]',
-    new_version: '2022.2.7',
+    region: 'us-west-2',
+    availability_zone: 'a,b',
+    base_versions: '["2025.1.10"]',
+    new_version: '2025.2.4',
```

Comment:
Latest available Scylla release is `2025.3.5`

Reply:
Originally I bumped the version up one major or minor step as the tests were badly unstable enough as they were (they still remain somewhat flaky), I asked in #engineering about the upgrade policy. I suppose we can do the latest version later, but right now I want to migrate the branch over with least possible unpleasant surprises.